Microsoft.Toolkit.Uwp.SampleApp/Models/Sample.cs

```diff
@@ -648,9 +648,9 @@ private static Type LookForTypeByName(string typeName)
                 }
             }
 
-            // Search in Microsoft.Toolkit.Uwp.UI.Controls
-            var controlsProxyType = GridSplitter.GridResizeDirection.Auto;
-            assembly = controlsProxyType.GetType().GetTypeInfo().Assembly;
+            // Search in Microsoft.Toolkit.Uwp.UI.Controls.Core
+            var controlsCoreProxyType = StackMode.Replace;
+            assembly = controlsCoreProxyType.GetType().GetTypeInfo().Assembly;
```

Comment:
Actually was just thinking we could create a list at the top like:

```cs
// Assemblies to search for types from
var assemblies = new List<Assembly>() {
   StackMode.Replace.GetType().GetTypeInfo().Assembly, // Microsoft.Toolkit.Uwp.UI.Controls.Core
   ... 
};

foreach (var assembly in assemblies)
{
    foreach (var typeInfo in assembly.ExportedTypes)
    {
        if (typeInfo.Name == typeName)
        {
             return typeInfo;
        }
    }
}
```

Would save some code. We may also want to think about a cache or something in the future, but we can do that in the re-write of the sample app in the future.